packages/cli/src/rollup-plugins/typescript-declarations/create-generator.ts

```diff
@@ -178,26 +178,28 @@ export async function createDeclarationCreator(
       searchDeps(new Set(resolvedEntrypointPaths));
       return allDeps;
     },
-    getDeclarationFile: async (
+    getDeclarationFiles: async (
       filename: string
-    ): Promise<{ name: string; content: string }> => {
+    ): Promise<{ name: string; content: string }[]> => {
```

Comment:
since there are concerns about how this gets consumed later, maybe we should refactor this to return:
```ts
interface DeclarationFile {
  name: string;
  content: string;
}

Promise<{ types: DeclarationFile, map?: DeclarationFile }>
```